src/ios/CDVStatusBar.m

```diff
@@ -132,13 +132,15 @@ - (void)pluginInitialize
 
     // blank scroll view to intercept status bar taps
     self.webView.scrollView.scrollsToTop = NO;
-    UIScrollView *fakeScrollView = [[UIScrollView alloc] initWithFrame:UIScreen.mainScreen.bounds];
+    
+    CGRect bounds = [self.viewController.view.window bounds];
```

Comment:
Probably on pluginInitialize the viewController isn't initialized yet and that's why you get (0,0)x(0,0)

I think we can keep the UIScreen.mainScreen.bounds here as the fakeScrollView is a non visible UIScrollView that is bigger than the actual screen, and that is still true on multitasking mode.